### PR TITLE
Have fun with code

### PR DESCRIPTION
**Pull request by rasbt**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
 x = 1
-y = x * 2
+y = x * 2 * 3
```
Comment (williamFalcon) [inline main.py]:
```suggestion
y = x * 2 * 3 * 5
```

Comment (williamFalcon) [reply main.py]: hey! thanks for the pull request. I think we should also multiply by 5

Comment (rasbt) [reply main.py]: How about just replacing `2*3*5` by `30`? What do you think?

Comment (williamFalcon) [reply main.py]: that sounds great. go for it!

**Pull request merged**